How-To-Create-A-Pull-Request-for-Free-Code-Camp.md

```diff
@@ -0,0 +1,72 @@
+# Overview
+### What is a Pull Request?
+A pull request (PR) is a method of submitting proposed changes to the Free Code Camp Repo (or any Repo, for that matter).  You will make changes to copies of the files which make up Free Code Camp in a personal fork, then apply to have them accepted by Free Code Camp proper.
+
+### Methods
+There are two methods of creating a Pull for Free Code Camp:
+
+1. Editing files via the GitHub Interface
+1. Editing files on a local clone
+
+# Important: ALWAYS EDIT ON A BRANCH
+Take away only one thing from this document, it should be this:  Never, EVER make edits to the `Staging` branch.  ALWAYS make a new branch BEFORE you edit files.  This is critical, becuase if your PR is not accepted, your copy of `staging` will be forever sullied and the only way to fix it is to delete your fork and re-fork.
```

Comment:
Maybe it's worth it if it gets people to always be using branches, but I'm somewhat uncomfortable with making this sound so dire since recovering a staging branch is as simple as `git reset --hard HEAD~numOfBadCommits` and then force pushing. I feel like all too often people get stuck in git and think the only option is to delete and restart when it rarely is the only option.